wordpress/app.yaml

```diff
@@ -0,0 +1,80 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
```

Comment:
we need pv as well right to bound it to pvc ?

Reply:
I don't think so. When the PVC is created Longhorn will create the PV under the hood and the PVC is bounded to it.
![image](https://user-images.githubusercontent.com/18577298/94545732-37fd0c00-024d-11eb-8290-f133439d7ab5.png)

Reply:
![image](https://user-images.githubusercontent.com/8190114/94546045-faa77700-026a-11eb-920b-6c06fe0aa466.png)
I was trying out on a new cluster with Longhorn installed and I see this

Reply:
ok my cluster had some problems, so it was having issues creating the Pv's. This looks good. Will merge

Reply:
I'm glad to read that, not your problems with the cluster, but the merge. :wink:

Reply:
in few minutes it would be live I would want you to test it on a fresh new cluster so that we can announce that ;)